app/forms/mappings/Formatters.scala

```diff
@@ -46,8 +46,9 @@ trait Formatters {
       override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], Int] =
         baseFormatter
           .bind(key, data)
-          .right.map(_.replace(",", ""))
-          .right.flatMap {
```

Comment:
This doesn't resolve this issue:
'in some scenarios date field validation is not picking up second missing entry in fields and tells users that only one. This might create difficulties for users when they will need to correct another field when error was mention only one.'

![image](https://user-images.githubusercontent.com/53828896/236166502-7b93c1ec-a7a3-40b7-a6f4-7f7171411e9d.png)

Reply:
Also need to address this issue:
'when one of the date fields is empty error message misleading users saying that date should be real, when actual error should be 'The date must include a year' (A)'
![image](https://user-images.githubusercontent.com/53828896/236166793-89df6ec4-2ad3-4529-81f0-c6adf905ac5a.png)